both/collections/competition.js

```diff
@@ -37,7 +30,17 @@ Competitions.attachSchema({
       afFieldInput: {
         type: "timeOfDayMinutes"
       }
-    }
+    },
+    custom: function() {
+      var obj = validationObject(this, ['calendarStartMinutes']);
+      var error = null;
+      ScheduleEvents.find({competitionId: this.docId}).forEach(function(event) {
```

Comment:
Note to self/question: does this validation fire when new ScheduleEvents are added/change?